source/documentation/getting-started.html.md

```diff
@@ -1052,6 +1052,88 @@ Result:
 ManageIQ refreshes the details for the chosen cloud providers.
 
 
+## Containers
+
+Currently the container provider types that is currently possible to add are:
```

Comment:
Used `currently` twice and there is a grammatical mistake of is vs are.  Suggest
`The currently supported container provider types are:`